src/t8_forest/t8_forest_cxx.cxx

```diff
@@ -978,11 +974,11 @@ t8_forest_element_face_normal (t8_forest_t forest, t8_locidx_t ltreeid, const t8
      *    |      |
      *    x ---- x 0
      *
-     *   Since V,C in R^3, we need N perpendicular to V and N in space (C,V)
+     *   Since V,C in R^3, we need N perpendicular to V and C in space (C,V)
      *   This N is given by N = C - <C,V>/<V,V> V
      *   <.,.> being the dot product.
      *   Since in general the corner is not 0, we consider the affine problem
-     *   with corner vectoy V_a and V_b, and shift it by -V_a.
```

Comment:
Eagle eye 🦅

Reply:
I do not know why, but my typos script caught that